rbac/management/workspace/view.py

```diff
@@ -99,7 +97,8 @@ def update(self, request, *args, **kwargs):
 
     def partial_update(self, request, *args, **kwargs):
         """Patch a workspace."""
-        payload = json.loads(request.body or "{}")
+        self.validate_workspace(request, "patch")
+        payload = request.data or "{}"
         for field in payload:
```

Comment:
Should the other end of `or` still be a string `"{}"` or should it be a dictionary literal `{}`?

Reply:
Looks like it should be a `{}` if I understand the intent. Confirmed with this test:

```
    def test_partial_update_empty(self):
        """Test for updating a workspace with empty body."""
        workspace_data = {
            "name": "New Workspace",
            "description": "New Workspace - description",
            "tenant_id": self.tenant.id,
        }

        workspace = Workspace.objects.create(**workspace_data)

        url = reverse("v2_management:workspace-detail", kwargs={"pk": workspace.id})
        client = APIClient()
        response = client.patch(url, {}, format="json", **self.headers)
        self.assertEqual(response.status_code, status.HTTP_200_OK)
        self.assertEqual(response.get("content-type"), "application/json")
        data = response.data
        self.assertEqual(data.get("name"), "New Workspace")
        self.assertEqual(data.get("description"), "New Workspace - description")
        self.assertEqual(data.get("id"), str(workspace.id))
        self.assertNotEquals(data.get("created"), "")
        self.assertNotEquals(data.get("modified"), "")
        self.assertEquals(data.get("type"), "standard")

        update_workspace = Workspace.objects.filter(id=workspace.id).first()
        self.assertEquals(update_workspace.name, "New Workspace")
```

Reply:
Good catch, thanks! Updated w/test.